src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -97,11 +112,15 @@ export const NavBarSchedule: FC<NavBarScheduleProps> = ({ isVisible }) => {
           className={classes}
           onClick={onWeekdayClick}
         >
-          {dayIndex === 0 ? "Today" : format(day, "E")}
+          {hasScheduleStartDate
+            ? format(day, "E, LLL d")
+            : dayIndex === 0
+            ? "Today"
+            : format(day, "E")}
```

Comment:
```suggestion
          {formatDayLabel(day)}
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1419/commits/b78373170c83bbc19324c8694b04ea35b9ab06ef